ob-pattern-1/confs/obkm/repository/conf/identity/application-authentication.xml

```diff
@@ -0,0 +1,225 @@
+            <Parameter name="enrolUserInAuthenticationFlow">true</Parameter>
+            <Parameter name="usecase">local</Parameter>
+            <Parameter name="secondaryUserstore">primary</Parameter>
+            <Parameter name="TOTPAuthenticationEndpointURL">totpauthenticationendpoint/totp.jsp</Parameter>
+            <Parameter name="TOTPAuthenticationEndpointErrorPage">totpauthenticationendpoint/totpError.jsp</Parameter>
+            <Parameter name="TOTPAuthenticationEndpointEnableTOTPPage">totpauthenticationendpoint/enableTOTP.jsp</Parameter>
+            <Parameter name="redirectToMultiOptionPageOnFailure">false</Parameter>
+        </AuthenticatorConfig-->
+    </AuthenticatorConfigs>
+
+    <!--
+		Sequences allow specifying authentication flows for different
+		registered applications. \'default\' sequence is taken if an
+		application specific sequence doesn't exist in this file or
+		in the Application Mgt module.
+    -->
+    <Sequences>
+        <!-- Default Sequence. This is mandatory -->
+        <Sequence appId="default">
+            <Step order="1">
+                <Authenticator name="BasicAuthenticator"/>
+            </Step>
+        </Sequence>
+    </Sequences>
+
+    <!--
+            AuthenticationEndpointQueryParams are the request parameters
+            that would be sent to the AuthenticationEndpoint.
+            'action' defines the behaviour: if 'include', only the defined
+            parameters would be included in the request.
+            If 'exclude' specified, all the parameters received by the
+            Authentication Framework would be sent in the request except
+            the ones specified.
+            'sessionDataKey', 'type', 'relyingParty', 'sp' and 'authenticators'
+            parameters will be always sent. They should not be specified here.
+        -->
+    <AuthenticationEndpointQueryParams action="exclude">
+        <AuthenticationEndpointQueryParam name="username"/>
+        <AuthenticationEndpointQueryParam name="password"/>
+        <AuthenticationEndpointQueryParam name="SAMLRequest"/>
+    </AuthenticationEndpointQueryParams>
+
+    <!-- Required in OB -->
+    <!-- Make removeOnConsumeFromAPI="false" -->
+    <AuthenticationEndpointRedirectParams action="include" removeOnConsumeFromAPI="true">
+        <AuthenticationEndpointRedirectParam name="sessionDataKeyConsent"/>
+        <AuthenticationEndpointRedirectParam name="relyingParty"/>
+        <AuthenticationEndpointRedirectParam name="authenticators"/>
+        <AuthenticationEndpointRedirectParam name="authFailureMsg"/>
+        <AuthenticationEndpointRedirectParam name="authFailure"/>
+    </AuthenticationEndpointRedirectParams>
+
+    <!--TenantDomainDropDownEnabled>true</TenantDomainDropDownEnabled>
+    <TenantDataListenerURLs>
+      <TenantDataListenerURL>/authenticationendpoint/tenantlistrefresher.do</TenantDataListenerURL>
+    </TenantDataListenerURLs-->
+
+</ApplicationAuthentication>
```

Comment:
@kdanW shall we place an EOF line here?

Reply:
Fixed with https://github.com/wso2/kubernetes-open-banking/pull/13/commits/a6c289bdb04eb294570ae486704103047497e89d